saplings/register-login/src/index.ts

```diff
@@ -34,6 +34,16 @@ interface FormEventHandler {
   (this: HTMLFormElement, event: Event): void;
 }
 
+interface Canopy {
```

Comment:
Is this really necessary here? I think it makes more sense as part of the splinter ui app itself, where $CANOPY is initially defined in `splinter-ui/app/src/App.js`